cg/server/api.py

```diff
@@ -113,24 +97,25 @@ def families():
         records = db.cases_to_mip_analyze()
         count = len(records)
     else:
-        customer_obj = None if g.current_user.is_admin else g.current_user.customer
+        customer_objs = None if g.current_user.is_admin else g.current_user.customers
         case_query = db.families(
             enquiry=request.args.get("enquiry"),
-            customer=customer_obj,
+            customers=customer_objs,
             action=request.args.get("action"),
         )
         count = case_query.count()
         records = case_query.limit(30)
+
     data = [case_obj.to_dict(links=True) for case_obj in records]
     return jsonify(families=data, total=count)
 
 
 @BLUEPRINT.route("/families_in_customer_group")
 def families_in_customer_group():
     """Fetch families in customer_group."""
-    customer_obj = None if g.current_user.is_admin else g.current_user.customer
+    customer_objs = None if g.current_user.is_admin else g.current_user.customers
     families_q = db.families_in_customer_group(
```

Comment:
type hint and perhaps a more descriptive variable name, `family_query_result` perhaps?